server/views/pages/work.njk

```diff
@@ -1,6 +1,8 @@
 {% extends 'layout/default.njk' %}
 
-{% block pageMeta %}{% endblock %}
+{% block pageMeta %}
+  <link rel="canonical" href="{{ pageConfig.organization.url }}{{ work.urlWithoutQuery }}" />
```

Comment:
Could we just set `next.wellcomecollection.org` here, as there's no reason this should be tied to an abstracted out Org?